ambry-server/src/integration-test/java/com.github.ambry.server/ServerHardDeleteTest.java

```diff
@@ -69,6 +67,10 @@
   private MockTime time;
   private AmbryServer server;
   private MockClusterMap mockClusterMap;
+  private ArrayList<BlobProperties> properties;
```

Comment:
This file as such doesn't have any new additions. But initially I was looking if I can add any tests here for hard delete pause and resume and refactored it. Later realized it was easy via IndexTest. So leaving the refactored code as is. (Instead of looking at the diff, look at the raw file and it might be easy to follow the test)